src/test/resources/docker-compose.yml

```diff
@@ -0,0 +1,91 @@
+version: '2.2'
+services:
+  es01:
+    image: "docker.elastic.co/elasticsearch/elasticsearch:${ELASTICSEARCH_VERSION}"
```

Comment:
Any reason not to use the latest version, which I think is around `3.7`?

Reply:
No reason other than it was the version used in Elastic's [documented example](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html). I can change it to the latest and commit if the tests pass afterward.